test/tasks/common_test.go

```diff
@@ -56,7 +56,13 @@ func checkFileContent(t *testing.T, wsDir, filename, want string) {
 
 func checkFileExists(t *testing.T, filename string) {
 	if _, err := os.Stat(filename); os.IsNotExist(err) {
-		t.Fatal(err)
+		t.Fatalf("Want %s, but got nothing", filename)
+	}
+}
+
+func checkFilesExist(t *testing.T, wantFiles []string, wsDir string) {
```

Comment:
@henninggross If we flip around parameter order of `wantFiles` and `wsDir` we could make `wantFiles` a vararg. With that, `checkFilesExist` would work for 0 to n files, we could even consider inlining `checkFileExists` into `checkFilesExist` then. What's your thoughts on that @michaelsauter?

Reply:
@henninggross I just gave the approach a try in commit 4f97367bf21e88929187bbf365c1a4388eeb5c3f